WordPress/wordpress-playground
# Add core-pr and gutenberg-pr Query API parameters

Enables easy mixing of WordPress and Gutenberg Pull Requests via Query
API parameters. For example:

https://playground.wordpress.net/?core-pr=6883&gutenberg-pr=65337

These parameters work even when a Blueprint is specified and take
precedence over the WordPress version specified by the Blueprint

 ## Testing instructions

1. Go to http://localhost:5400/website-server/?core-pr=6883&gutenberg-pr=65337
2. Confirm the WordPress release is a development version (in wp-admin, bottom right
   corner)
3. Confirm the Gutenberg plugin is installed## Motivation for the change, related issues
4. Confirm the Gutenberg PR previewer at http://localhost:5400/website-server/gutenberg.html still works
5. Confirm the WordPress PR previewer at http://localhost:5400/website-server/wordpress.html still works
